test/service/EvidenceUploadService.test.ts

```diff
@@ -0,0 +1,69 @@
+import { expect } from 'chai';
+import * as FormData from 'form-data';
+import nock = require('nock');
+
+import { EvidenceUploadService } from 'app/service/EvidenceUploadService'
+
+describe('EvidenceUploadService', () => {
+
+    const KEY: string = 'mockKey';
+    const HOST: string = 'http://localhost';
+    const evidenceUploadService = new EvidenceUploadService(HOST , KEY);
```

Comment:
Service name could be made more generic as in the future it will likely also have file download capability. When renamed it could have been moved to separate PR and merged as a first bit so that Francisco could expand on it by adding download function.